webpack.config.js

```diff
@@ -61,16 +63,27 @@ module.exports = [{
           presets: ['es2015', 'react'],
         },
       },
+
+      /*
+       * Parse SCSS to minified CSS, then postprocess with postcss autoprefixer plugin
+       */
       {
         test: /\.scss$/,
-        loader: ExtractTextPlugin.extract(['css','sass']),
+        loader: ExtractTextPlugin.extract('style', ['css?minimize', 'postcss', 'sass']),
       },
+
+      /*
+       * Load in files with CSS alone
+       */
       {
         test: /\.(png|woff|woff2|eot|ttf|svg)$/,
         loader: 'url-loader?limit=10000',
       },
     ],
   },
+  postcss: function () {
+    return [autoprefixer({browsers: ['last 2 versions']})];
```

Comment:
It's notation from `autoprefixer`. It means it will `autoprefix` such that everything should work on the last two versions of all major browsers.